src/meta/src/barrier/context/recovery.rs

```diff
@@ -745,451 +766,50 @@ impl GlobalBarrierWorkerContextImpl {
             cdc_table_snapshot_split_assignment,
         }))
     }
-}
-
-impl GlobalBarrierWorkerContextImpl {
-    // Migration timeout.
-    const RECOVERY_FORCE_MIGRATION_TIMEOUT: Duration = Duration::from_secs(300);
 
-    /// Migrate actors in expired CNs to newly joined ones, return true if any actor is migrated.
-    async fn migrate_actors(
+    async fn load_stream_actors(
         &self,
-        active_nodes: &mut ActiveStreamingWorkerNodes,
-    ) -> MetaResult<HashMap<DatabaseId, HashMap<TableId, InflightStreamingJobInfo>>> {
-        let mgr = &self.metadata_manager;
-
-        // all worker slots used by actors
-        let all_inuse_worker_slots: HashSet<_> = mgr
-            .catalog_controller
-            .all_inuse_worker_slots()
-            .await?
-            .into_iter()
-            .collect();
-
-        let active_worker_slots: HashSet<_> = active_nodes
-            .current()
+        all_info: HashMap<DatabaseId, HashMap<TableId, InflightStreamingJobInfo>>,
```

Comment:
Seems no need to pass an owned info. Reference is enough, and then the clone on the caller is unnecessary.